editoast/src/views/timetable.rs

```diff
@@ -449,21 +458,46 @@ impl Conflict {
 }
 
 #[derive(Debug, Clone, PartialEq, Deserialize, Serialize, ToSchema)]
-struct PacedTrainOccurrenceId {
-    paced_train_id: i64,
-    index: u64,
+#[serde(untagged)]
+enum PacedTrainOccurrenceId {
+    Occurrence {
+        paced_train_id: i64,
+        index: u64,
+    },
+    Exception {
+        paced_train_id: i64,
+        exception_key: String,
+        index: Option<u64>,
+    },
 }
 
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, PartialEq)]
+/// This ID is used to identify paced train occurrences and exceptions when sending them to the core API for conflict detection.
 enum TrainId {
+    /// Id format `{id}`
     TrainSchedule(i64),
+    /// Id format `{paced_train_id}#{index}`
     PacedTrainOccurrence { paced_train_id: i64, index: u64 },
+    /// Id format `{paced_train_id}@{exception_key}@{index}`
+    PacedTrainException {
+        paced_train_id: i64,
+        exception_key: String,
+        index: Option<u64>,
+    },
 }
 
 impl Display for TrainId {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         match self {
             Self::TrainSchedule(id) => write!(f, "{id}"),
+            Self::PacedTrainException {
+                paced_train_id,
+                exception_key,
+                index,
+            } => {
+                let index = index.map(|i| i.to_string()).unwrap_or_default();
+                write!(f, "{paced_train_id}@{exception_key}@{index}")
```

Comment:
Shouldn't this be the following instead, in order to be coherent with the format of an index in a regular occurrence?
```suggestion
                write!(f, "{paced_train_id}@{exception_key}#{index}")
```

Reply:
`#` for occurrences and `@` for exceptions, it coherent 😂

Reply:
Is it only humor and you agree with my point, or you disagree... and therefore, I don't see the argument. I can't tell 🤔

Reply:
Sorry for the humor, i don't agree with you, I use # for occurrences and @ for exceptions to clearly differentiate them in the string format - it’s intentional and helps parsing.

Could you explain why you think this format is not coherent?

Reply:
The `#` symbol is often associated with an index. And also, having a different prefix symbol for identifying different semantical information feels more intuitive (`@{exception_key}` and `#{index}`). What do you think? Maybe we need some other opinions (cc @younesschrifi @leovalais)

Reply:
It’s just a separator — the symbol itself isn’t semantically tied to the content (also during parsing). Initially, it was a very simple way to transfer structured data via a string.
But as our needs have evolved and become slightly more complex, perhaps it’s time to consider a more robust or explicit format for exchanging data?

Reply:
strong agree on keeping the `#` for indexes, less confusing, and the # sign already has that meaning anyway